pinot-core/src/main/java/com/linkedin/pinot/core/segment/store/SegmentLocalFSDirectory.java

```diff
@@ -75,7 +75,34 @@ public Path getPath() {
 
   @Override
   public long getDiskSizeBytes() {
-    return FileUtils.sizeOfDirectory(segmentDirectory.toPath().toFile());
+    // [PINOT-3479] For newly added refresh segments, the new segment will
```

Comment:
Do we really want to ?
It doesn't add value to outside users but not leaking any data.

-Adwait

On Fri, Aug 12, 2016 at 4:32 PM, Ravi Aringunram <notifications@github.com>
wrote:

> In pinot-core/src/main/java/com/linkedin/pinot/core/segment/
> store/SegmentLocalFSDirectory.java
> <https://github.com/linkedin/pinot/pull/387#discussion_r74674253>:
>
> > @@ -75,7 +75,34 @@ public Path getPath() {
> >
> >    @Override
> >    public long getDiskSizeBytes() {
> > -    return FileUtils.sizeOfDirectory(segmentDirectory.toPath().toFile());
> > +    // [PINOT-3479] For newly added refresh segments, the new segment will
>
> Remove the JIRA tag here.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/linkedin/pinot/pull/387/files/cee758b5dfdde4c02e478f321de62de078990ccb#r74674253>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABAmRsG-FtG6Je_U1RDUP2NPwMEcIlg1ks5qfQKpgaJpZM4Jjibk>
> .
>